src/app/components/task/CreateTaskMenu.js

```diff
@@ -51,14 +40,27 @@ class CreateTaskMenu extends React.Component {
   };
 
   render() {
+    const isTask = this.props.fieldset === 'tasks';
+    const { teamSettings } = this.props;
+    const buttonProps = {};
+
+    if (teamSettings) {
+      buttonProps.color = 'primary';
+      buttonProps.variant = 'contained';
+    }
+
     return (
       <div>
-        <StyledCreateTaskButton
+        <Button
           className="create-task__add-button"
           onClick={this.handleClick.bind(this)}
+          {...buttonProps}
         >
-          <FormattedMessage id="tasks.addTask" defaultMessage="Add task" />
-        </StyledCreateTaskButton>
+          { isTask ?
+            <FormattedMessage id="tasks.addTask" defaultMessage="New task" /> :
+            <FormattedMessage id="tasks.addMetadata" defaultMessage="New metadata" />
+          }
```

Comment:
That's fine for now... but important to know that it will work only while we have only two fieldsets.